### PR TITLE
tests: update static content shell version

### PR DESCRIPTION
**Pull request by brendankenny**
ref. https://github.com/GoogleChrome/lighthouse/issues/12209#issuecomment-793188171

### CODE REVIEW

File: .github/workflows/devtools.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
           ${{ env.BLINK_TOOLS_PATH }}
           ${{ github.workspace }}/lighthouse/.tmp/chromium-web-tests/content-shells
         # The number is how many times this hash key was manually updated to break the cache.
-        key: ${{ runner.os }}-0-${{ hashFiles('third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/*.js', 'lighthouse/.github/workflows/devtools.yml', 'lighthouse/lighthouse-core/test/chromium-web-tests/*', 'lighthouse/clients/devtools-entry.js', 'lighthouse/clients/devtools-report-assets.js', 'lighthouse/build/build-bundle.js', 'lighthouse/build/build-dt-report-resources.js') }}
+        key: ${{ runner.os }}-1-${{ hashFiles('third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/*.js', 'lighthouse/.github/workflows/devtools.yml', 'lighthouse/lighthouse-core/test/chromium-web-tests/*', 'lighthouse/clients/devtools-entry.js', 'lighthouse/clients/devtools-report-assets.js', 'lighthouse/build/build-bundle.js', 'lighthouse/build/build-dt-report-resources.js') }}
```
Comment (connorjclark) [inline .github/workflows/devtools.yml:36]: let's add `third-party/download-content-shell/download-content-shell.js` here

Comment (brendankenny) [reply .github/workflows/devtools.yml:36]: > let's add

Is that the royal "us" or are you asking me to add that here? :)

Comment (connorjclark) [reply .github/workflows/devtools.yml:36]: haha "royal us", I haven't heard of that

yes, please add that file in this hash thingy

**Comment by brendankenny** [comment]
Currently at 48 minutes for smoke-devtools and no more log messages. Not looking great for this as a fix :/

**Review by adamraine — APPROVED** [review]
In case you still want to merge

**Pull request closed without merging**